### PR TITLE
Release notes for Operator 2.2.0.

### PR DESCRIPTION
**Pull request by cstevandy**
TBD release date, but the context is ready to be reviewed

### CODE REVIEW

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md [hunk 1]
```diff
@@ -13,6 +13,16 @@ For information on the current status of deployment to Mendix for Private Cloud
...
 
+### December ???, 2021
+
+#### Mendix Operator v2.2.0 and Mendix Gateway Agent v2.2.0
+
+* To improve the security score of all Mendix for Private Cloud images, we replaced a deprecated [operator-sdk v0.18.2](https://v0-18-x.sdk.operatorframework.io/docs/) dependency with a maintained version of [kubebuilder](https://book.kubebuilder.io/).
+  * This update allows us to address [CVE-2020-8565](https://access.redhat.com/security/cve/cve-2020-8565), [CVE-2020-26160](https://access.redhat.com/security/cve/cve-2020-26160) and [CVE-2020-29652](https://access.redhat.com/security/cve/cve-2020-29652). These CVEs do not affect previously released versions Mendix for Private Cloud.
+  * This is an internal library change and doesn't change the behavior of any Mendix for Private Cloud components.
+* When switching *Review and Apply* screen in `mxpc-cli`, previous results will be cleared.
+* We have fixed an issue where attempting to apply a custom TLS configuration in non-interactive mode with `mxpc-cli` failed with a panic error.
```
Comment (MarkvanMents) [inline content/releasenotes/developer-portal/mendix-for-private-cloud.md:24]: What do you mean by a 
> panic error

is this something that customers will recognize?

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md [hunk 2]
```diff
@@ -13,6 +13,16 @@ For information on the current status of deployment to Mendix for Private Cloud
 
 ## 2021
 
+### December ???, 2021
+
+#### Mendix Operator v2.2.0 and Mendix Gateway Agent v2.2.0
+
+* To improve the security score of all Mendix for Private Cloud images, we replaced a deprecated [operator-sdk v0.18.2](https://v0-18-x.sdk.operatorframework.io/docs/) dependency with a maintained version of [kubebuilder](https://book.kubebuilder.io/).
```
Comment (MarkvanMents) [inline content/releasenotes/developer-portal/mendix-for-private-cloud.md:20]: Have you checked with Frank about this discussion of security?

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md [hunk 3]
```diff
@@ -13,6 +13,16 @@ For information on the current status of deployment to Mendix for Private Cloud
...
+  * This update allows us to address [CVE-2020-8565](https://access.redhat.com/security/cve/cve-2020-8565), [CVE-2020-26160](https://access.redhat.com/security/cve/cve-2020-26160) and [CVE-2020-29652](https://access.redhat.com/security/cve/cve-2020-29652). These CVEs do not affect previously released versions Mendix for Private Cloud.
```
Comment (MarkvanMents) [inline content/releasenotes/developer-portal/mendix-for-private-cloud.md:21]:
```suggestion
  * This update allows us to address [CVE-2020-8565](https://access.redhat.com/security/cve/cve-2020-8565), [CVE-2020-26160](https://access.redhat.com/security/cve/cve-2020-26160) and [CVE-2020-29652](https://access.redhat.com/security/cve/cve-2020-29652). These CVEs do not affect previously released versions of Mendix for Private Cloud.
```

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md [hunk 4]
```diff
@@ -13,6 +13,16 @@ For information on the current status of deployment to Mendix for Private Cloud
...
+  * This is an internal library change and doesn't change the behavior of any Mendix for Private Cloud components.
```
Comment (MarkvanMents) [inline content/releasenotes/developer-portal/mendix-for-private-cloud.md:22]:
```suggestion
  * This internal library change does not change the behavior of any Mendix for Private Cloud components.
```

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md [hunk 5]
```diff
@@ -13,6 +13,16 @@ For information on the current status of deployment to Mendix for Private Cloud
...
+* When switching *Review and Apply* screen in `mxpc-cli`, previous results will be cleared.
```
Comment (MarkvanMents) [inline content/releasenotes/developer-portal/mendix-for-private-cloud.md:23]: Not sure I understand exactly what this is. Would
```suggestion
* When switching to the *Review and Apply* screen in `mxpc-cli`, previous results will be cleared.
```
cover it?

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Hi Christian (@cstevandy )
I can't actually edit this pull request at all. It may be because you have made a PR from Dmitrii's fork and so GitHub won't allow me any access at all.
Can you look at the questions/comments below and see if you can commit the changes and respond to the other questions.
Otherwise, I may have to merge this into my own branch so that I can edit it.
Thanks for the PR.

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md (hunk 1 above)
Comment (zlogic) [reply content/releasenotes/developer-portal/mendix-for-private-cloud.md:24]: Yes, this was based on an error report from a customer (unfortunately there's no support ticket number that we can reference)

Comment (MarkvanMents) [reply content/releasenotes/developer-portal/mendix-for-private-cloud.md:24]: OK, We can leave it there if customers are going to recognize it!

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md (hunk 2 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/mendix-for-private-cloud.md:20]: Approved by Frank by email.

**Review by MarkvanMents — APPROVED** [review]
Looks good.
I'll fix the dates when I publish it, so we don't have to worry about the editing issue.
I need to add something to our featured page anyway.
Let me know when you are ready to publish.
Mark

**Comment by zlogic** [comment]
Thank you @MarkvanMents, can you publish these release notes now?

**Pull request merged**